sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -818,36 +818,24 @@ class CastSuite extends SparkFunSuite with ExpressionEvalHelper {
       "interval 1 years 3 months -3 days")
   }
 
-  test("cast string to boolean") {
-    checkCast("true", true)
-    checkCast("tru", true)
-    checkCast("tr", true)
+  test("cast string to boolean with Spark dialect") {
```

Comment:
Shall we remove `with Spark dialect` since we can distinguish with package name?